src/Radar.js

```diff
@@ -39,9 +40,8 @@ export default class Radar extends Viz {
       },
       Path: {}
     });
+    this._size = accessor("size");
     this._value = accessor("value");
```

Comment:
You seem to be using `this._value` throughout where you should now be using `this._size`

Reply:
Because `this._value` is used for accessing to the measure.

Reply:
Ahhh I see, I'm sorry I got confused. `this._value` is for the actual number along the axis, and `this._size` is used for each "spoke" of the radar.

In that case, we should rename `this._size` to be `this._metric` 😬